Prace_domowe/Praca_domowa1/Grupa2/Agata_Kaczmarek/pd_1_Agata_Kaczmarek.ipynb

```diff
@@ -0,0 +1,1657 @@
+{
```

Comment:
<p>Bardzo dobrze że to opisałaś. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>